ui/src/components/LoginForm/index.tsx

```diff
@@ -38,29 +42,43 @@ const Login: React.FC<FormikProps<LoginFormValues>> = (props) => {
       </button>
     );
 
+  const handleModal = () => {
+    setModalOpen(false);
+  };
+
   return (
-    <form
-      className={cx('login-form', { 'has-error': !props.isValid })}
-      onSubmit={props.handleSubmit}>
-      <div className={cx('fields-container')}>
-        <div className={cx('2-step-swiper', cx(formStep))}>
-          <TextField
-            className={cx('email')}
-            placeholder="> enter your work email"
-            type="email"
-            name="email"
-          />
+    <div>
+      <div className={cx('login-form-container')}>
+        <form
+          className={cx('login-form', { 'has-error': !props.isValid })}
+          onSubmit={props.handleSubmit}>
+          <div className={cx('fields-container')}>
+            <div className={cx('2-step-swiper', cx(formStep))}>
+              <TextField
+                className={cx('email')}
+                placeholder="> enter your work email"
+                type="email"
+                name="email"
+              />
 
-          <div className={cx('password')}>
-            <span className={cx('back-icon')} onClick={gotoStep1} />
+              <div className={cx('password')}>
+                <span className={cx('back-icon')} onClick={gotoStep1} />
```

Comment:
Looks like this isn't formatted properly.

Reply:
ok